assets/tasks/common.json

```diff
@@ -110,10 +110,11 @@
                 "-nop",
                 "${workspaceFolder}/.conf/createDockerComposeProduction.ps1",
                 "${workspaceFolder}",
-                "${command:docker_registry}",
+                "${command:docker_login}",
                 "${command:docker_tag}",
+                "${command:docker_registry}",
                 "__container__",
-                "${command:torizon_gpu}"
+                "${config:torizon_gpu}"
```

Comment:
Why? Changing this to `config:` will not show the input for the user 🤔

Reply:
Yes, that is expected for the GPU configuration, which should be automatically filled base on the default board. This was leading to the problem of it being showed if the user is not using the `-vivante` GPU, which confused the user

Reply:
Also, it was different from the other places where it is used
![image](https://github.com/toradex/vscode-torizon-templates/assets/83607022/1be39f30-e621-435c-be0a-16d9e15bfa44)